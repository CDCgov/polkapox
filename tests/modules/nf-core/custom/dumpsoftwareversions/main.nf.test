nextflow_process {

    name "Test Process CUSTOM_DUMPSOFTWAREVERSIONS"
    script "modules/nf-core/custom/dumpsoftwareversions/main.nf"
    process "CUSTOM_DUMPSOFTWAREVERSIONS"

    test("Custom/DumpSoftwareVersions Test") {

        when {
            params {
            }
            process {
                """
                 input[0] = file("${projectDir}/tests/nf-test-data/collated_versions.yml")
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
