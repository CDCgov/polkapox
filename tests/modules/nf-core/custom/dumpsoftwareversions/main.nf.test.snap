{
    "Custom/DumpSoftwareVersions Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "2": [
                    
                ],
                "mqc_yml": [
                    
                ],
                "versions": [
                    
                ],
                "yml": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-07T14:43:11.277440394"
    }
}